<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnSetStart">
        <property name="text">
         <string>设置起点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetEnd">
        <property name="text">
         <string>设置终点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearPath">
        <property name="text">
         <string>清除路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGenerate">
        <property name="text">
         <string>生成迷宫</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNextPath">
        <property name="text">
         <string>下一条路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFindPath">
        <property name="text">
         <string>找到路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnShortest">
        <property name="text">
         <string>找到最短路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>加载迷宫</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>9</width>
        <height>9</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>